src/scripts/game.js

```diff
@@ -207,6 +238,34 @@ class Game {
       this.beehiveSparkles.splice(this.beehiveSparkles.indexOf(object), 1);
     }
   }
+
+  updateHealth(points) {
+    this.health += points;
+    if (this.health <= 0) this.bee.capture();
+    if (this.health > 100) this.health = 100;
```

Comment:
I think these conditionals are mutually exclusive so you should use an `else if`. (I.e. you wouldn't want to have the bee.capture and also set health to 100.

Reply:
Good call, may help catch a weird edge case where bee is mid-pollination but also under attack.